### PR TITLE
chore: set docker syntax using erda registry

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
set docker syntax using erda registry

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=612943&iterationID=12783&tab=BUG&type=BUG)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  set docker syntax using erda registry            |
| 🇨🇳 中文    |    更改默认dockerfile的syntax          |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: build/dockerfiles/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# syntax = docker/dockerfile:1.2
+# syntax = registry.erda.cloud/retag/dockerfile:1.2
```
Comment (iutx) [inline build/dockerfiles/Dockerfile:1]: It seems feasible to upgrade to version 1.8.

**Review by iutx — COMMENTED** [review]
@chengjoey

File: build/dockerfiles/Dockerfile (hunk 1 above)
Comment (chengjoey) [reply build/dockerfiles/Dockerfile]: thanks, i set it to latest

`# syntax = registry.erda.cloud/retag/dockerfile:latest`

**Review by sfwn — APPROVED** [review]

**Pull request merged**